src/core/Translator.test.js

```diff
@@ -0,0 +1,38 @@
+import Core from '../../src/core/index.js'
+import russian from '../../src/locales/ru_RU.js'
```

Comment:
should we be using `import` here if everywhere else we are using `require`? IT probably works because Babel, and maybe its fine, but thought I’d bring it up.